pkg/wallet/account.go

```diff
@@ -98,6 +98,10 @@ func (a *Account) SignTx(t *transaction.Transaction) error {
 	if a.privateKey == nil {
 		return errors.New("account is not unlocked")
 	}
+	if len(a.Contract.Parameters) == 0 {
```

Comment:
If not then we don't know how to create an invocation script?

Reply:
Yes, we don't yet have a proper CLI and haven't discussed what cases to support.